src/java.base/share/classes/java/security/Security.java

```diff
@@ -892,6 +896,9 @@ public static String getProperty(String key) {
      * @see java.security.SecurityPermission
      */
     public static void setProperty(String key, String datum) {
+        if (SecPropLoader.isInclude(key)) {
+            return;
+        }
```

Comment:
Don't you want to throw an exception here?

Reply:
May be. This is a public API that only documents `SecurityException` for cases in which there is a Security Manager and `NullPointerException` for cases in which either the key or the value are `null`. Wouldn't be the exact case here, unless we stretch it a bit and document a new type of unchecked exception. I was thinking of `IllegalArgumentException`. What do you think?

Reply:
Proposed a change here: https://github.com/openjdk/jdk/pull/16483/commits/530bd81c2a73a39f00c6ba9524752f30600cd04d

Reply:
I like this.

Just wondering, shall we throw IAE if `getProperty("include")` is called as well? In a very rare case that someone is already using it in `java.security` file and the value happens to be an existing file name, then this will catch it when the user is trying to read it.

Reply:
Yes, we can do that.

Reply:
Done in 580d34a63b5d3b6c7d2323413f338527db2d9acd.

Reply:
Thanks. You might need to update the CSR to include this in the spec change.

Reply:
Yes, I updated the user guide and the CSR to reflect this behavior.